product_docs/docs/pgd/5/manual/01-provisioning-hosts.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Provisioning Hosts
+---
+
+## Provisioning hosts
+
+The first step in the process of deploying PGD is to provision and configure hosts.
+
+You can deploy to virtual machine instances in the cloud with Linux installed, on-premise virtual machines with Linux installed or on-premise physical hardware also with Linux installed. 
+
+Whichever [supported Linux operating system](https://www.enterprisedb.com/resources/platform-compatibility#bdr) and whichever deployment platform you select, the result of provisioning a machine should be a Linux system that can be accessed by you using SSH.
+
+Each machine provisioned should be able to make connections to any other machine you are provisioning for your cluster. 
+On cloud deployments, this may be done over the public network or over a VPC. 
+On-premise deployments should be able to connect over the local network.
+
+!!! Note Cloud provisioning guides
+
+If you are new to cloud provisioning, these guides may provide assistance:
+
+  Vendor | Platform | Guide
+  ------ | -------- | ------
+  Amazon | AWS      | [Tutorial: Get started with Amazon EC2 Linux instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html)
+  Microsoft | Azure | [Quickstart: Create a Linux virtual machine in the Azure portal](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal?tabs=ubuntu)
+  Google | GCP | [Create a Linux VM instance in Compute Engine](https://cloud.google.com/compute/docs/create-linux-vm-instance)
+
+!!!
+
+## Configuring hosts
+
+### Create an admin user
+
+We recommend that you configure an admin user for each provisioned instance. 
+The admin user should be configured for SSH access using certificates. 
+The admin user should have sudo privileges.
+
+### Ensure networking connectivity
+
+With the admin user created, ensure that each machine can communicate with the other machines you are provisioning.
+
+## Worked example
+
+For the example in this section, we have provisioned three hosts with Red Hat Enterprise Linux 9.
+
+host-one
+host-two
+host-three
```

Comment:
Please change that into a bulleted or numbered list, otherwise the names are printed next to each other.

Reply:
Done